cirq/experiments/fidelity_estimation.py

```diff
@@ -0,0 +1,70 @@
+"""Estimation of fidelity associated with experimental circuit executions."""
+
+from typing import Iterable, Tuple, Set
+
+import numpy as np
+
+from cirq.circuits import Circuit
+from cirq.ops import QubitOrder, QubitOrderOrList
+from cirq.sim import final_wavefunction
+
+
+def compute_linear_xeb_fidelity(
```

Comment:
The word `compute` is redundant; drop it. Just `linear_xeb_fidelity`.

Reply:
Or e.g. `linear_xeb_fidelity_of_samples`

Reply:
Agree with dropping "compute". I wouldn't append "of_samples" since it's equally redundant: samples are the only type of input that XEB fidelity estimator supports (as is explained in the comments).